pkg/metrics/metrics.go

```diff
@@ -56,6 +56,10 @@ var (
 		Name: "aws_account_operator_pool_size_vs_unclaimed",
 		Help: "Report the difference between the pool size and the number of unclaimed account CRs",
 	}, []string{"name"})
+	metricTotalAccountPendingVerification = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "aws_account_operator_total_aws_account_pending_verification",
+		Help: "Report how many accounts are waiting for limit increase in AWS",
```

Comment:
Does it not wait for the enterprise support too?

Reply:
Yes, both. Will update the metric description

Reply:
updated